docs/_writing_dependencies_conda.rst

```diff
@@ -301,18 +302,79 @@ general Bioconda_ is usually the correct starting point.
     been accepted into Bioconda_ - including tools for image analysis,
     natural language processing, and cheminformatics.
 
-At this time, the most relevant source for information on building Conda packages for Galaxy
-is probably the Bioconda_ documentation - in particular check out the `contributing documentation
-<https://bioconda.github.io/contributing.html>`__.
+To get quickly learn to write Conda_ recipes for typical Galaxy tools,
+please read the following pieces of external documentation.
+
+- `Contributing to Bioconda <https://bioconda.github.io/contributing.html>`__ in particular focusing on
+
+  - `One time setup <https://bioconda.github.io/contrib-setup.html>`__
+  - `Contributing a recipe <https://bioconda.github.io/contribute-a-recipe.html>`__ (through "Write a Recipe")
+- `Building Conda Package <https://conda.io/docs/building/bpp.html#>`__ in particular
+
+  - `Building conda packages with conda skeleton <https://conda.io/docs/build_tutorials/pkgs.html>`__
```

Comment:
Maybe highlight with a single sentence that skeletons are the usual way to create templates for scripting languages ... so that people don't start to write this by hand?